src/Bicep.Core.Samples/Files/TestFramework_CRLF/main.diagnostics.bicep

```diff
@@ -112,4 +125,5 @@ test testRuntime 'samples/runtime.bicep' = {}
 // Test inexistent file
 
 test testInexistentFile 'samples/inexistent.bicep' = {}

```

Comment:
You may want to move this and the `testResource2` test symbol to the `InvalidTestFramework_CRLF` baseline, since baselines that don't start with `Invalid` should compile successfully.